product-service/serverless.ts

```diff
@@ -1,39 +1,92 @@
-import type { AWS } from '@serverless/typescript';
+import type {AWS} from '@serverless/typescript';
 
 import getProductsList from '@functions/get-products-list';
 import getProductById from '@functions/get-product-by-id';
 import createProduct from '@functions/create-product';
+import catalogBatchProcess from '@functions/catalog-batch-process';
 
 const serverlessConfiguration: AWS = {
-  service: 'product-service',
-  frameworkVersion: '2',
-  custom: {
-    webpack: {
-      webpackConfig: './webpack.config.js',
-      includeModules: true,
+    service: 'product-service',
+    frameworkVersion: '2',
+    custom: {
+        webpack: {
+            webpackConfig: './webpack.config.js',
+            includeModules: true,
+        },
     },
-  },
-  plugins: ['serverless-webpack'],
-  provider: {
-    name: 'aws',
-    runtime: 'nodejs14.x',
-    region: 'us-east-1',
-    apiGateway: {
-      minimumCompressionSize: 1024,
-      shouldStartNameWithService: true,
+    plugins: ['serverless-webpack'],
+    provider: {
+        name: 'aws',
+        runtime: 'nodejs14.x',
+        region: 'us-east-1',
+        apiGateway: {
+            minimumCompressionSize: 1024,
+            shouldStartNameWithService: true,
+        },
+        environment: {
+            AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+            PG_HOST: '***.rds.amazonaws.com',
+            PG_PORT: '5432',
+            PG_DATABASE: 'mydb',
+            PG_USERNAME: '***',
+            PG_PASSWORD: '***',
```

Comment:
how do you hide these values? Does it work like this?

Reply:
In AWS I committed with the real data. But it is a bad practice to store the credentials in the repo